components/header/Buttons.tsx

```diff
@@ -72,12 +71,15 @@ function CartButton() {
     >
       <div class="indicator">
         {totalItems && (
-          <span class="indicator-item badge badge-secondary badge-sm">
+          <span
+            style={{ top: "-12px" }}
+            class=" absolute  indicator-item badge badge-secondary badge-lg h-6 w-6 pa  bg-white  text-black border-none font-extralight"
```

Comment:
Mantem a doc do tailwind aberta full time nesse começo para se acostumar com as propriedades. A grande maioria ja existe no Tailwind e se nao existir da para criar as classes: https://tailwindcss.com/docs/top-right-bottom-left
```suggestion
            class="-top-3.5 absolute indicator-item badge badge-secondary badge-lg h-6 w-6 pa  bg-white text-black border-none font-extralight"
```